packages/web/src/builders/package/package.impl.ts

```diff
@@ -114,15 +121,26 @@ export function run(
             filesize()
           ];
 
+          const globals = options.globals
+            ? options.globals.reduce((acc, item) => {
+                acc[item.moduleId] = item.global;
+                return acc;
+              }, {})
+            : {};
           const entryFileTmpl = `${options.outputPath}/${context.target.project}.<%= extension %>.js`;
+          const externalPackages = dependencies
```

Comment:
External packages can be:
1. A workspace (buildable) project 
2. A custom external package as specified by builder option.
3. A dependency from `package.json`.